### PR TITLE
document systems and health pages per #598

### PR DESCRIPTION
**Pull request by john-floren-gravwell**

### CODE REVIEW

File: gui/gui.md [hunk 1]
```diff
@@ -97,3 +97,52 @@ The "Advanced Preferences" section can be ignored by most users. Selecting "Deve
...
+
+![](systems-and-health.png)
+
+### Storage, Indexers, & Wells
+
+This page shows information about the data stored in the indexers of the Gravwell system.
+
+![](storage.png)
+
+The Storage section shows a summary of how much data is in the system, with separate stats for hot and cold storage. The "dropper" graphic indicates how fast new entries are entering the system.
```
Comment (ashnwade) [inline gui/gui.md:113]: "The Storage section shows a summary of how much data is in the system, with separate stats for hot and cold storage"
Do we need the comma before `with`?

**Review by ashnwade — CHANGES_REQUESTED** [review]
The Gravwell GUI page provides a list of other pages for easy navigation, but the only parts of the GUI included in this page directly are the parts of the system that are always visible (main menu, notifications, account).
image

Everything else that you navigate to through the main menu has its own separate page and a link in the GUI page:
image

In this case, I think that Systems & Health should be added to the navigation menu and split off into its own separate page.

File: gui/gui.md (hunk 1 above)
Comment (john-floren-gravwell) [reply gui/gui.md:113]: i think it's correct

**Comment by john-floren-gravwell** [comment]
ptal

**Review by ashnwade — APPROVED** [review]
LGTM

**Pull request merged**